### PR TITLE
Add price server to docker compose

### PR DESCRIPTION
**Pull request by dolcalmi**
- Add price server to dev environment (docker compose)
- Add trigger to Make file
- Update .envrc with price env variables

### CODE REVIEW

**Review by jrbuhl93 — DISMISSED** [review]

File: docker-compose.yml [hunk 1]
```diff
@@ -1,9 +1,14 @@
 version: "2"
 services:
+  price:
+    build: https://github.com/GaloyMoney/price.git#main
```
Comment (bodymindarts) [inline docker-compose.yml:4]: This should be a tagged image not `build: ...#main`. This file is not for developing the images but to provide deps _as they are in production_ to develop _this_ app.

Comment (nicolasburtey) [reply docker-compose.yml:4]: we are not doing versioning yet on the price pod

Comment (dolcalmi) [reply docker-compose.yml:4]: is there a way to resolve it from chart repository(gcr.io/galoyapp/price)? or it is different?

Comment (nicolasburtey) [reply docker-compose.yml:4]: > is there a way to resolve it from chart repository(gcr.io/galoyapp/price with latest tag)? or is it different?

yes you can use the hash. should be shown in the build done by github action

Comment (dolcalmi) [reply docker-compose.yml:4]: I tried with `build: gcr.io/galoyapp/price:latest` and `build: gcr.io/galoyapp/price#latest` but is not working

**Review by samerbuna — APPROVED** [review]

**Review by bodymindarts — APPROVED** [review]

**Pull request merged**